NAMESPACE

```diff
@@ -71,6 +71,7 @@ S3method(heightDetails,titleGrob)
 S3method(heightDetails,zeroGrob)
 S3method(interleave,default)
 S3method(interleave,unit)
+S3method(knit_print,animint)
```

Comment:
hi @lazycipher this commit fixes the Rmd rendering problems for me. does it work for you?

Reply:
Yes! It's fixed now!